.prettierignore

```diff
@@ -0,0 +1,7 @@
+dist
+node_modules
+.yarn
+build
+coverage
+.docusaurus
+applications/xcm-domain-service.md
```

Comment:
Sorry for the delay here. What is the reason for adding applications/xcm-domain-service.md here?

Reply:
When I tested hooks, I found the format of this doc will cause some compile errors. So I decided to add this into ignore.

```
Compiled with problems:
×
ERROR in ./applications/xcm-domain-service.md
Module build failed (from ./node_modules/@docusaurus/mdx-loader/lib/index.js):
SyntaxError: /opt/src/w3f/Grants-Program/applications/xcm-domain-service.md: Expected corresponding JSX closing tag for <>. (46:94)

  44 | `}</em>{` Does XCM support calls between different smart contract languages? Like ink! `}<>{` Solidity.
  45 | `}<em parentName="p">{` What does a cross-chain domain service via XCM actually mean?
> 46 | `}</em>{` How can XCM be used to achieve "cross-parachain" domain-to-domain asset transfers?`}</p>
     |                                                                                               ^
  47 | <pre><code parentName="pre" {...{}}>{`* How is the gas & registration/renewal fee payment handled?
  48 |
  49 | * Is XCM needed for cross-chain domain ↔ address resolving?
    at instantiate (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:63:32)
    at constructor (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:358:12)
    at JSXParserMixin.raise (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:3255:19)
    at JSXParserMixin.jsxParseElementAt (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:6922:14)
    at JSXParserMixin.jsxParseElementAt (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:6900:32)
    at JSXParserMixin.jsxParseElementAt (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:6900:32)
    at JSXParserMixin.jsxParseElement (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:6957:17)
    at JSXParserMixin.parseExprAtom (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:6969:19)
    at JSXParserMixin.parseExprSubscripts (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10890:23)
    at JSXParserMixin.parseUpdate (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10873:21)
    at JSXParserMixin.parseMaybeUnary (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10849:23)
    at JSXParserMixin.parseMaybeUnaryOrPrivate (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10687:61)
    at JSXParserMixin.parseExprOps (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10692:23)
    at JSXParserMixin.parseMaybeConditional (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10669:23)
    at JSXParserMixin.parseMaybeAssign (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10630:21)
    at /opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10600:39
    at JSXParserMixin.allowInAnd (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12280:12)
    at JSXParserMixin.parseMaybeAssignAllowIn (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10600:17)
    at JSXParserMixin.parseParenAndDistinguishExpression (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:11513:28)
    at JSXParserMixin.parseExprAtom (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:11164:23)
    at JSXParserMixin.parseExprAtom (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:6974:20)
    at JSXParserMixin.parseExprSubscripts (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10890:23)
    at JSXParserMixin.parseUpdate (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10873:21)
    at JSXParserMixin.parseMaybeUnary (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10849:23)
    at JSXParserMixin.parseMaybeUnaryOrPrivate (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10687:61)
    at JSXParserMixin.parseExprOps (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10692:23)
    at JSXParserMixin.parseMaybeConditional (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10669:23)
    at JSXParserMixin.parseMaybeAssign (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10630:21)
    at JSXParserMixin.parseExpressionBase (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10584:23)
    at /opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10580:39
    at JSXParserMixin.allowInAnd (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12275:16)
    at JSXParserMixin.parseExpression (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:10580:17)
    at JSXParserMixin.parseReturnStatement (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12999:28)
    at JSXParserMixin.parseStatementContent (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12617:21)
    at JSXParserMixin.parseStatementLike (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12584:17)
    at JSXParserMixin.parseStatementListItem (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12564:17)
    at JSXParserMixin.parseBlockOrModuleBlockBody (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13185:61)
    at JSXParserMixin.parseBlockBody (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13178:10)
    at JSXParserMixin.parseBlock (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13166:10)
    at JSXParserMixin.parseFunctionBody (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:11947:24)
    at JSXParserMixin.parseFunctionBodyAndFinish (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:11933:10)
    at /opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13314:12
    at JSXParserMixin.withSmartMixTopicForbiddingContext (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12257:14)
    at JSXParserMixin.parseFunction (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13313:10)
    at JSXParserMixin.parseFunctionStatement (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12980:17)
    at JSXParserMixin.parseStatementContent (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12610:21)
    at JSXParserMixin.parseStatementLike (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12584:17)
    at JSXParserMixin.parseModuleItem (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:12561:17)
    at JSXParserMixin.parseBlockOrModuleBlockBody (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13185:36)
    at JSXParserMixin.parseBlockBody (/opt/src/w3f/Grants-Program/node_modules/@babel/parser/lib/index.js:13178:10)
```